packer/http/ks.cfg

```diff
@@ -1,46 +1,60 @@
-install
-cdrom
+# Version=EL8
+# Partition clearing information
+clearpart --all --initlabel
+autopart --type=lvm
+# Use network installation
+url --url http://mirror.steadfastnet.com/centos/8-stream/BaseOS/x86_64/os/
+# Keyboard layouts
+keyboard --vckeymap=us --xlayouts='us'
+# System language
 lang en_US.UTF-8
-keyboard us
-unsupported_hardware
-network --bootproto=dhcp --device=eth0 --onboot=on --hostname=centos7-katello-devel-stable.example.com
-rootpw vagrant
+
+# Network information
+network  --bootproto=dhcp --device=enp0s3  --activate --hostname=centos8-katello-devel-stable.example.com
 firewall --disabled
 selinux --permissive
-timezone UTC
-unsupported_hardware
-bootloader --location=mbr
-text
-skipx
-zerombr
-clearpart --all --initlabel
-autopart
-auth --enableshadow --passalgo=sha512 --kickstart
+# Root password
+rootpw vagrant
+# Run the Setup Agent on first boot
 firstboot --disabled
 eula --agreed
+# Do not configure the X Window System
+text
+skipx
+# System services
 services --enabled=NetworkManager,sshd
+# System timezone
+timezone UTC
 reboot
-user --name=vagrant --plaintext --password vagrant --groups=vagrant,wheel
 
-%packages --ignoremissing --excludedocs
-@Base
-@Core
-openssh-clients
-sudo
-openssl-devel
-readline-devel
-vim
-wget
+%packages --ignoremissing
```

Comment:
Why did you drop `--excludedocs`? Is that gone in EL8? And I'd drop `--ignoremissing`.